source/postprocess/velocity_boundary_statistics.cc

```diff
@@ -42,12 +42,15 @@ namespace aspect
                                         this->get_fe(),
                                         quadrature_formula,
                                         update_values |
+                                        update_JxW_values |
                                         update_q_points);
 
       std::vector<Tensor<1,dim> > velocities (fe_face_values.n_quadrature_points);
 
       std::map<types::boundary_id, double> local_max_vel;
       std::map<types::boundary_id, double> local_min_vel;
+      std::map<types::boundary_id, double> local_boundary_sqvel;
```

Comment:
this name is not quite clear to me, what about `local_velocity_square_integral`? That name is used in the `velocity_statistics` postprocessor. I would say that `boundary` is implied by the name of the class

Reply:
Done